rllib/core/rl_module/rl_module.py

```diff
@@ -130,22 +139,38 @@ def get_catalog(self) -> "Catalog":
         )
 
     def to_dict(self):
-        """Returns a serialized representation of the config."""
+        """Returns a serialized representation of the config.
```

Comment:
Do we need to store config_class_path? Do we want to allow users to extend RLModuleConfig? Do you have any example in mind?

Reply:
Hmm I was making the assumption that a user might make their own custom rl module config.

But if I'm wrong about that assumption I'll remove this.